README.md

```diff
@@ -178,13 +178,18 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-ruby@v1
-        with:
-          ruby-version: '2.6'
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - run: |
+          git fetch --prune --unshallow
```

Comment:
What's the benefit of passing in `--unshallow`? Asking because I usually set my linting pipelines up with a `--depth=N` (maybe 10 usually?) to reduce the size of the transfer and make the clone faster, because you're only linting a specific commit/snapshot in time

Reply:
@jamiew ```--depth=1``` didn't work for me and found ```--unshallow ``` resolve the issue without much very about depth size. I will create a wiki page and try to specify it over there.
Thanks for your input.

Reply:
@jamiew updating with
```
- run: |
          git fetch --no-tags --prune --depth=10 origin +refs/heads/*:refs/remotes/origin/*
```

Reply:
Did you mean to tag me in here?

Reply:
> Did you mean to tag me in here?

Oops! typo :exploding_head:

Reply:
@jaydorsey updated, please review.